biome.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "https://biomejs.dev/schemas/2.0.0/schema.json",
+  "files": {
+    "maxSize": 16000000,
+    "includes": ["{packages,lib,scripts,tests}/**/*.{mjs,js,ts,json}", "!**/dist-{cjs,es,types}/**", "!**/ruleset.ts"]
+  },
+  "formatter": {
```

Comment:
I've turned many rules off initially because we need to establish that biome is runnable in our build systems before reintroducing rules that would lead to code changes.